<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="wsc-042" name="Test1" tests="9" failures="4" timestamp="23 Jan 2017 13:11:45 GMT" time="0.053" errors="0">
  <testcase name="checkSimpleSubtraction" time="0.007" classname="com.CalculatorTest.SubtractionTest">
    <failure type="java.lang.AssertionError" message="2 - 3 operation result is wrong! expected [-1.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: 2 - 3 operation result is wrong! expected [-1.0] but found [0.0]
	at com.CalculatorTest.SubtractionTest.checkSimpleSubtraction(SubtractionTest.java:44)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- checkSimpleSubtraction -->
  <testcase name="checkSubtractionFloats" time="0.001" classname="com.CalculatorTest.SubtractionTest"/>
  <testcase name="checkSubtractionNegative" time="0.001" classname="com.CalculatorTest.SubtractionTest"/>
  <testcase name="checkSubtractionSelf" time="0.002" classname="com.CalculatorTest.SubtractionTest">
    <failure type="java.lang.AssertionError" message="1.1 - 1.1 operation result is wrong! expected [0.0] but found [1.1]">
      <![CDATA[java.lang.AssertionError: 1.1 - 1.1 operation result is wrong! expected [0.0] but found [1.1]
	at com.CalculatorTest.SubtractionTest.checkSubtractionSelf(SubtractionTest.java:57)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- checkSubtractionSelf -->
  <testcase name="checkZeroSubtraction" time="0.001" classname="com.CalculatorTest.SubtractionTest">
    <failure type="java.lang.AssertionError" message="3 - 0 operation result is wrong! expected [3.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: 3 - 0 operation result is wrong! expected [3.0] but found [0.0]
	at com.CalculatorTest.SubtractionTest.checkZeroSubtraction(SubtractionTest.java:30)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- checkZeroSubtraction -->
  <testcase name="checkAdditionFloats" time="0.001" classname="com.CalculatorTest.AdditionTest"/>
  <testcase name="checkAdditionNegative" time="0.001" classname="com.CalculatorTest.AdditionTest"/>
  <testcase name="checkSimpleAddition" time="0.001" classname="com.CalculatorTest.AdditionTest">
    <failure type="java.lang.AssertionError" message="2 + 3 operation result is wrong! expected [5.0] but found [2.0]">
      <![CDATA[java.lang.AssertionError: 2 + 3 operation result is wrong! expected [5.0] but found [2.0]
	at com.CalculatorTest.AdditionTest.checkSimpleAddition(AdditionTest.java:44)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- checkSimpleAddition -->
  <testcase name="checkZeroAddition" time="0.001" classname="com.CalculatorTest.AdditionTest"/>
</testsuite> <!-- Test1 -->
